examples/react-navigation/src/StackNavigator.test.js

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { screen, fireEvent } from '@testing-library/react-native';
+import { renderWithNavigation } from './test-utils';
+import StackNavigator from './StackNavigator';
+
+test('Home screen contains the header and list of items', async () => {
```

Comment:
```suggestion
test('Home screen contains the header and list of items', () => {
```
there is no await in the test